SpaceDash/View/CustomView/CustomPopUpView.swift

```diff
@@ -0,0 +1,177 @@
+        didSet{
+            closeBtn.addTarget(delegate, action: #selector(RocketViewController.closeBtnTappped), for: .touchUpInside)
+        }
+    }
+    
+    ///Demo data
+    let imgArr = ["test_image", "test_image" , "test_image"]
+    
+    ///  Collectionview for Multiple images
+    lazy var collectionView:UICollectionView = {
+        let layout:UICollectionViewFlowLayout = UICollectionViewFlowLayout.init()
+        layout.scrollDirection = .horizontal
+        let cv = UICollectionView(frame: CGRect.zero, collectionViewLayout: UICollectionViewLayout.init())
+        cv.translatesAutoresizingMaskIntoConstraints = false
+        cv.showsHorizontalScrollIndicator = false
+        cv.register(ImageCollectionViewCell.self, forCellWithReuseIdentifier: "ImageCollectionViewCell")
+        cv.setCollectionViewLayout(layout, animated: false)
+        cv.delegate = self
+        cv.dataSource = self
+        cv.isPagingEnabled = true
+        cv.backgroundColor = .white
+        cv.bounces = false
+        return cv
+    }()
+    
+    /// Rocket Description
+    public let descriptionText:UILabel = {
+        let l = UILabel()
+        l.translatesAutoresizingMaskIntoConstraints = false
+        l.numberOfLines = 0
+        l.textColor = .black
+        l.text = "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged."
+        return l
+    }()
+    
+    /// Page controlls
+    private let pageControl:UIPageControl = {
+        let c = UIPageControl()
+        c.translatesAutoresizingMaskIntoConstraints = false
+        c.currentPage = 0
+        return c
+    }()
+    
+    /// Close the modal
+    private let closeBtn:UIButton = {
+        let btn = UIButton()
+        btn.setBackgroundImage(UIImage(systemName: "xmark.circle.fill")?.withRenderingMode(.alwaysTemplate), for: .normal)
+        btn.tintColor = .white
+        btn.translatesAutoresizingMaskIntoConstraints = false
+        return btn
+    }()
+    
+    /// Divider Line
+    private let dividerLine:UIView = {
+        let v = UIView()
+        v.translatesAutoresizingMaskIntoConstraints = false
+        v.backgroundColor = UIColor(white: 0, alpha: 0.2)
+        return v
+    }()
+    
+    /// Wikepedia btn
+    private let wekipediaBtn:UIButton = {
+        let btn = UIButton()
+        btn.translatesAutoresizingMaskIntoConstraints = false
+        btn.setTitle("Wekipedia", for: .normal)
+        btn.setTitleColor(.systemBlue, for: .normal)
+        btn.titleLabel?.font = UIFont.systemFont(ofSize: 16, weight: .medium)
+        return btn
+    }()
+    
+    override init(frame: CGRect) {
+        super.init(frame:frame)
+        backgroundColor = .white
```

Comment:
We are using a custom colour named DashCream for white. Use that colour instead of white everywhere.